src/IO/IO.jl

```diff
@@ -0,0 +1,6 @@
+module IO
```

Comment:
Following discussion in the standup meeting, lets rename this to be `InputOutput`